spaces/domains.json

```
@@ -1,4 +1,5 @@
 {
+  "snapshot.radworks.org": "radicle.eth",
```

I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/radicle.eth
```suggestion
  "snapshot.radworks.org": "radicle.eth",
```

---

I am not trying to create a new space, I'm just trying to change the slug on my space (which I've renamed). This is the space:

https://snapshot.org/#/gov.radicle.eth

I'd like it to be gov.radworks.eth instead of gov.radicle.eth

---

@abbey-titcomb You'll need to create a new space with the ENS gov.radworks.eth and then request a space migration. Please follow the process here https://docs.snapshot.org/user-guides/spaces/migrate-or-delete#migrate-a-space